src/main/java/me/jellysquid/mods/sodium/client/render/SodiumWorldRenderer.java

```diff
@@ -392,6 +392,11 @@ public boolean isEntityVisible(Entity entity) {
             }
         }
 
+        // Ensure entities with the glowing effect are not culled, since they are always visible to the player.
+        if (entity.isGlowing()) {
```

Comment:
this check should run before the actual culling code

Reply:
Thanks for the feedback! However, I'm not sure if the order of visibility checks should matter much in this situation.

Putting the glowing visibility check earlier would allow entities to skip the chunk visibility check if they are glowing (since the method returns earlier), however there is no measurable performance improvement from doing so as checking for chunk visibility is not at all demanding. Even in ideal conditions (>1000 entities, all glowing), there is no measurable difference from making the change on my system.

I had placed this check at the end because it seems to be the least likely to happen. Usually, I try to order from most likely to least likely, which should allow the method to return earlier (this almost always provides no performance benefit- but I like to do it anyways :P)

Actually, now that I've thought about this a bit more, you may have mistaken this method for being something like `shouldCullEntity()` rather than `isEntityVisible()`, in which case you would be correct and the order does matter. However, since we only want to determine if the entity is actually visible (by returning true anywhere in the method), putting it before the chunk visibility check should not be necessary.

I could be completely missing something else here though, since I am relatively new to programming in Java. If so please let me know.